### ISSUE TITLE
Sonic Robo Blast 2 Kart

### ISSUE THREAD

**Issue by eroc1990** [CubeCoders/AMPTemplates#1119]
# Module Request

Note: Please try setting up a configuration yourself before raising an issue to request a configuration:  ~~https://config.getamp.sh/~~
***There is a newer beta version available at https://iceofwraith.github.io/GenericConfigGen/

## Game Information:

Game Title: Sonic Robo Blast 2 Kart
How to obtain: https://github.com/STJr/Kart-Public/releases (Win). Docker repos available from third parties. PPA out of date for Linux installations.
Suppoted OSs: Windows (officially), anything running Docker (unofficially)
Supports RCON/Console input?: Unknown

## I confirm:

- [x] that I have searched for an existing module request for this application.
- [x] that the server is an official server distributed by the games publishers/developers.
- [x] that the server is available to the general public.
- [x] that the server can be run headless (without a GUI).
- [x] I have already attempted to create a configuration myself using the configuration tool

**Comment by Hacksawfred3232** [comment]
I know you were waiting for that template to be merged. However, I decided to no longer work on this due to disagreement of views between myself and the CubeCoders staff. I would look into how to compile the game yourself on Linux and how to run it inside a docker container.

My apologizes.

**Comment by IceOfWraith** [comment]
> I know you were waiting for that template to be merged. However, I decided to no longer work on this due to disagreement of views between myself and the CubeCoders staff. I would look into how to compile the game yourself on Linux and how to run it inside a docker container.
> 
> My apologizes.

My memory is foggy. What was the disagreement?

**Comment by Hacksawfred3232** [comment]
> My memory is foggy. What was the disagreement?

AI data centers vs actually employing workers. You were part of that discussion earlier.

**Comment by Greelan** [comment]
I've got more complete templates in my dev branch anyway (they have been there a long time). Only issue is AMP's log tailing

**Comment by IceOfWraith** [comment]
Ahh okay different usernames. Unrelated to the pull request. Lol

**Comment by IceOfWraith** [comment]
> I've got more complete templates in my dev branch anyway (they have been there a long time). Only issue is AMP's log tailing

What's the issue with this one? Is it the date on the filename one?

**Comment by Greelan** [comment]
No, AMP just stops tailing. Sometimes it's resolved by opening the log file in the file manager lol

**Comment by IceOfWraith** [comment]
I didn't realize it did that. I'll add a bug.

**Comment by Greelan** [comment]
Yes you did. Lol

**Comment by IceOfWraith** [comment]
> Yes you did. Lol

At one point yes. But I meant just now I didn't realize. XD

**Comment by IceOfWraith** [comment]
The log tailing is fixed in the coming 2.8.0.6 release.
